client/components/LayoutEditor/useLayoutPubs.ts

```diff
@@ -72,29 +80,43 @@ export const useLayoutPubs = (
 	);
 
 	useEffect(() => {
-		if (previousLayout && layout && previousLayout !== layout) {
+		const allowDuplicatePubsChanged = previousAllowDuplicatePubs !== allowDuplicatePubs;
```

Comment:
how are the previous duplicate pubs and duplicate pubs related? Is this saying only rerun the query if duplcate pubs list has changed?

Reply:
Yeah, this is saying "refresh the layout if the user changed the _allow duplicate Pubs_ setting"